pkg/pipeline/extract/aggregate/aggregate.go

```diff
@@ -167,20 +185,29 @@ func (aggregate Aggregate) Evaluate(entries []config.GenericMap) error {
 func (aggregate Aggregate) GetMetrics() []config.GenericMap {
 	var metrics []config.GenericMap
 	for _, group := range aggregate.Groups {
+		// TODO: remove prefixes when filtering is implemented in prom encode.
 		metrics = append(metrics, config.GenericMap{
-			"name":                               aggregate.Definition.Name,
-			"operation":                          aggregate.Definition.Operation,
-			"record_key":                         aggregate.Definition.RecordKey,
-			"by":                                 strings.Join(aggregate.Definition.By, ","),
-			"aggregate":                          string(group.normalizedValues),
-			"value":                              fmt.Sprintf("%f", group.value),
-			"recentRawValues":                    group.RecentRawValues,
-			"count":                              fmt.Sprintf("%d", group.count),
-			aggregate.Definition.Name + "_value": fmt.Sprintf("%f", group.value),
-			strings.Join(aggregate.Definition.By, "_"): string(group.normalizedValues),
+			"name":            aggregate.Definition.Name,
+			"operation":       aggregate.Definition.Operation,
+			"record_key":      aggregate.Definition.RecordKey,
+			"by":              strings.Join(aggregate.Definition.By, ","),
+			"aggregate":       string(group.normalizedValues),
+			"total_value":     fmt.Sprintf("%f", group.value),
```

Comment:
s/group.value/group.totalValue ... this will help to make sure that when reading the code you understand what value we are talking about.